### PR TITLE
Updating the backup and restore doc-velero

### PR DESCRIPTION
**Pull request by dockerymick**
https://app.zenhub.com/workspaces/engineering-backlog-do-not-delete-604fab62d4b98d00150a2854/issues/stolostron/backlog/22250

### CODE REVIEW

File: clusters/backup_and_restore.adoc [hunk 1]
```diff
@@ -88,6 +89,61 @@ The operator defines the `backupSchedule.cluster.open-cluster-management.io` res
...
+resources:
+ limits:
+   cpu: "1"
+   memory: 256Mi
+ requests:
+   cpu: 500m
+   memory: 128Mi
+----
+//what's considered a "regular" sccenario
+The limits from the previous sample work well with regular scenarios, but might need to be updated when your cluster backs up a large number of resources. For instance, when back up is run on a hub cluster that manages 2000 clusters, then the Velero pod crashes due to the out-of-memory error (OOM). The following configuration allows for the backup to complete for this scenario:
```
Comment (dockerymick) [inline clusters/backup_and_restore.adoc]: @birsanv

Comment (birsanv) [reply clusters/backup_and_restore.adoc]: "regular" sccenario = out of the box ( I don't have exact numbers here, and velero doesn't give this info )

**Review by birsanv — APPROVED** [review]
lgtm

File: clusters/backup_and_restore.adoc (hunk 1 above)
Comment (dockerymick) [reply clusters/backup_and_restore.adoc]: Got it, thanks for explaining and reviewing!

**Pull request merged**